### DISCUSSION TITLE
Is there support for drawing underlined text?

### DISCUSSION THREAD

**Discussion by eangelov** [SixLabors/Fonts#232] [Ideas] [answered]
Currently FontStyle holds values for Regular/Bold/Italic/BoldItalic is there any way to draw underlined text?

**Comment by JimBobSquarePants** [comment, marked as answer]
This isn't the correct place for questions about the [Fonts library](https://github.com/SixLabors/Fonts/discussions). I'm going to transfer you to the correct place.

**Comment by JimBobSquarePants** [comment]
No. See https://github.com/SixLabors/Fonts/issues/7 should you wish to contribute support.
